src/Display/Display.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <Arduino.h>
+#include <SPI.h>
+#include <Wire.h>
+#include <Adafruit_GFX.h>
+#include <Adafruit_SSD1306.h>
+#include "DisplayConsts.h"
+
+// Configure OLED
+#ifndef DISPLAY_ADDR
+  #define DISPLAY_ADDR 0x3C // Mosto common address used for Oled displays
+#endif
+class Display : public Adafruit_SSD1306
```

Comment:
Small suggestion: Let's add a spaceline in between.
```suggestion
#endif

class Display : public Adafruit_SSD1306
```